### ISSUE TITLE
Console log "Ractive.js: Failed to compute <...>"

### ISSUE THREAD

**Issue by woodlandhunter** [ractivejs/ractive#1924]
On edge, getting this message in the console for the fiddle below:

```
Ractive.js: Failed to compute "${current-akey-0-astring-slice(0,4)}"
<snip callstack>
```

Fiddle:

http://jsfiddle.net/a6v45t9m/

**Comment by Madgvox** [comment]
That is very strange. It only errors if all three of the `ractive.set()` calls are present.

**Comment by martypdx** [comment]
What happens is that the expression gets evaluated prior to the section fragment being removed.

I just tried with #1740 branch (`0.8`) where I put some effort into being smarter about these kinds of changes and there's no error - so looks like a won't fix for `0.7` - it's harmless other than the annoying console warn.

**Comment by Madgvox** [comment]
I'll close this issue then, as it looks like it'll get cleaned up automatically by the next release. Feel free to open a new issue if you run into any more problems.

**Comment by abacaj** [comment]
Waiting on this as I have the same message when section fragment is being removed and updated with a new set of items.

**Comment by abacaj** [comment]
Also I tried 0.8 edge, and noticed it breaks my ractive tabs component. The isActive function doesn't get called - I suppose the view is no longer re-eval'ing the function `{{isActive(i)}}`
